hack/make/.binary

```diff
@@ -47,6 +47,14 @@ if [ "$(go env GOOS)/$(go env GOARCH)" != "$(go env GOHOSTOS)/$(go env GOHOSTARC
 			export CC=x86_64-w64-mingw32-gcc
 			export CGO_ENABLED=1
 			;;
+		linux/arm)
+			export CC=arm-linux-gnueabi-gcc
```

Comment:
Did you want to detect further based on `go env GOARM` to choose `gnueabi` vs `gnueabihf`? (`5` vs `7`)

Reply:
Updated, wdyt?